catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/helper/MetacardHelper.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.registry.schemabindings.helper;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.xml.bind.JAXBElement;
+import javax.xml.bind.Marshaller;
+
+import org.apache.commons.io.IOUtils;
+import org.codice.ddf.parser.Parser;
+import org.codice.ddf.parser.ParserConfigurator;
+import org.codice.ddf.parser.ParserException;
+import org.codice.ddf.registry.schemabindings.EbrimConstants;
+
+import com.google.common.base.Charsets;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.AttributeImpl;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.RegistryObjectType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.RegistryPackageType;
+
+public class MetacardHelper {
+    private Parser parser;
+
+    private ParserConfigurator unmarshalConfigurator;
+
+    private ParserConfigurator marshalConfigurator;
+
+    public MetacardHelper(Parser parser) {
+        setParser(parser);
+    }
+
+    /**
+     * Converts the metacards metadata into a RegistryPackageType object
+     *
+     * @param mcard A metacard with ebrim metadata
+     * @return A RegistryPackageType object created from the ebrim metadata
+     * @throws ParserException
+     */
+    public RegistryPackageType getRegistryPackageFromMetacard(Metacard mcard)
+            throws ParserException {
+        InputStream inputStream = null;
+        JAXBElement<RegistryObjectType> registryObjectTypeJAXBElement = null;
+        try {
+            String metadata = mcard.getMetadata();
+            inputStream = new ByteArrayInputStream(metadata.getBytes(Charsets.UTF_8));
+            registryObjectTypeJAXBElement = parser.unmarshal(unmarshalConfigurator,
+                    JAXBElement.class,
+                    inputStream);
+        } finally {
+            IOUtils.closeQuietly(inputStream);
+        }
+
+        if (registryObjectTypeJAXBElement == null) {
+            throw new ParserException("Error parsing metacards xml as ebrim");
+        }
+        return (RegistryPackageType) registryObjectTypeJAXBElement.getValue();
```

Comment:
Should we check for a `null` value here and throw an exception instead of returning `null`?

If we need to return `null`, we need to add `@Nullable` and explain when `null` would be returned.